.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,244 @@
+          # Set health checks to wait until postgres has started
+          options: >-
+            --health-cmd pg_isready
+            --health-interval 2s
+            --health-timeout 5s
+            --health-retries 30
+          ports:
+            - 5432:5432
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)